tasks/pecl-install-library.yml

```diff
@@ -0,0 +1,49 @@
+---
+
+- name: PHP | Pecl check if library {{ item.name }} exists
+  shell: "set -o pipefail && {{ php_install_dir }}/bin/pecl list | grep {{ item.name }}"
+  args:
+    executable: /bin/bash
+  register: php_pecl_library_system_info
+  ignore_errors: true
+  changed_when: false
+
+- name: PHP | Set Initial Fact
+  set_fact:
+    php_pecl_library_installed: false
+    php_pecl_library_installed_version: ""
+
+- name: PHP | Set Fact for version
+  set_fact:
+    php_pecl_library_installed: true
+    php_pecl_library_installed_version: "{{ php_pecl_library_system_info.stdout_lines[0].split()[1] }}"
+  when: php_pecl_library_system_info.stdout | length > 0
+
+- name: PHP | Install {{ item.name }} library
+  shell: "set -o pipefail && printf '\n' | {{ php_install_dir }}/bin/pecl install {{ item.library }}-{{ item.version }}"
+  args:
+    executable: /bin/bash
+  when: php_pecl_library_installed is false
+
+- name: PHP | Upgrade/Downgrade {{ item.name }} library
+  block:
+    - name: PHP | Uninstall {{ item.name }} because actual version is {{ php_pecl_library_installed_version }}
+      shell: "set -o pipefail && printf '\n' | {{ php_install_dir }}/bin/pecl uninstall {{ item.library }}"
+      args:
+        executable: /bin/bash
+    - name: PHP | Install {{ item.name }} version {{ item.version }}
+      shell: "set -o pipefail && printf '\n' | {{ php_install_dir }}/bin/pecl install {{ item.library }}-{{ item.version }}"
+      args:
+        executable: /bin/bash
+  when: php_pecl_library_installed is true and php_pecl_library_installed_version != item.version
+
+- name: PHP | Check that the 99-{{ item.name }}_{{ item.type }} configuration file exists
```

Comment:
This task (and the last condition at line 49) is useless, because the templating function controls if the file exists or not and if there will be modifications in it.